sql/4_2_2-to-5_0_0_upgrade.sql

```diff
@@ -2137,3 +2137,167 @@ CREATE TABLE `product_registration` (
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES ('Eye_Defaults_for_GENERAL', 'LADNEXA', 'normal lacrimal gland and orbit', 91, 0, 0, '', 'EXT', '', 0, 0, 0, '');
 #EndIf
+
+#IfNotRow2D list_options list_id page_validation option_id therapy_groups_add#addGroup
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `notes`, `activity`) VALUES ('page_validation', 'therapy_groups_add#addGroup', '/interface/therapy_groups/index.php?method=addGroup', 120, '{group_name:{presence: true}}', 1);
+#EndIf
+
+#IfNotRow2D list_options list_id page_validation option_id therapy_groups_edit#editGroup
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `notes`, `activity`) VALUES ('page_validation', 'therapy_groups_edit#editGroup', '/interface/therapy_groups/index.php?method=groupDetails', 125, '{group_name:{presence: true}}', 1);
+#EndIf
+
+#IfNotRow2D list_options list_id page_validation option_id tg_add#add-participant-form
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `notes`, `activity`) VALUES ('page_validation', 'tg_add#add-participant-form', '/interface/therapy_groups/index.php?method=groupParticipants', 130, '{participant_name:{presence: true}, group_patient_start:{presence: true}}', 1);
+#EndIf
+
+#IfNotTable therapy_groups
+CREATE TABLE `therapy_groups` (
+  `group_id` int(11) NOT NULL auto_increment,
+  `group_name` varchar(255) NOT NULL ,
+  `group_start_date` date NOT NULL ,
+  `group_end_date` date,
+  `group_type` tinyint NOT NULL,
+  `group_participation` tinyint NOT NULL,
+  `group_status` int(11) NOT NULL,
+  `group_notes` text,
+  `group_guest_counselors` varchar(255),
+  PRIMARY KEY  (`group_id`)
+) ENGINE=InnoDB;
+#EndIf
+
+#IfNotTable therapy_groups_participants
+CREATE TABLE `therapy_groups_participants` (
+  `group_id` int(11) NOT NULL,
+  `pid` int(11) NOT NULL ,
+  `group_patient_status` int(11) NOT NULL,
+  `group_patient_start` date NOT NULL ,
+  `group_patient_end` date,
+  `group_patient_comment` text,
+  PRIMARY KEY (`group_id`,`pid`)
+) ENGINE=InnoDB;
+#EndIf
+
+#IfNotTable therapy_groups_participant_attendance
+CREATE TABLE `therapy_groups_participant_attendance` (
+  `form_id` int(11) NOT NULL,
+  `pid` int(11) NOT NULL ,
+  `meeting_patient_comment` text ,
+  `meeting_patient_status` varchar(15),
+  PRIMARY KEY (`form_id`,`pid`)
+) ENGINE=InnoDB;
+#EndIf
+
+#IfNotTable therapy_groups_counselors
+CREATE TABLE `therapy_groups_counselors`(
+	`group_id` int(11) NOT NULL,
+	`user_id` int(11) NOT NULL,
+	PRIMARY KEY (`group_id`,`user_id`)
+) ENGINE=InnoDB;
+#EndIf
+
+#IfMissingColumn openemr_postcalendar_events pc_gid
+ALTER TABLE openemr_postcalendar_events ADD pc_gid int(11) DEFAULT 0 AFTER pc_pid;
+#EndIf
+
+#IfNotRow2D list_options list_id lists option_id groupstat
```

Comment:
For seq on lists. Best to start at 10 and increment by 10. This then allows room to add items down the road.